cmp/grid/columns/Column.ts

```diff
@@ -424,6 +431,7 @@ export class Column {
     fieldPath: Some<string>;
     colId: string;
     isTreeColumn: boolean;
+    isLeftZoneColumn: boolean;
```

Comment:
can we just set isTreeColumn in our ZonedGrid implementation ? Or otherwise just recognize this is the first column in a hierarchical zoned grid?

Would like to avoid

Reply:
thanks for addressing this!